annif/cli.py

```diff
@@ -122,12 +122,15 @@ def run_list_projects():
     List available projects.
     """
 
-    template = "{0: <25}{1: <45}{2: <8}"
-    header = template.format("Project ID", "Project Name", "Language")
+    template = "{0: <25}{1: <45}{2: <10}{3: <7}"
+    header = template.format(
+        "Project ID", "Project Name", "Language", "Trained")
     click.echo(header)
     click.echo("-" * len(header))
     for proj in annif.project.get_projects(min_access=Access.private).values():
-        click.echo(template.format(proj.project_id, proj.name, proj.language))
+        click.echo(
+            f'{proj.project_id: <25}{proj.name: <45}{proj.language: <10}'
+            f'{str(proj.is_trained): <7}')
```

Comment:
I didn't find a better way to use the same field widths as in the header but this. Formatting to the template could have been better?

Reply:
You're right, using the template is nicer because then you avoid declaring the column widths twice. Maybe I got carried away with my suggestion to use f-strings...